code/__DEFINES/research.dm

```diff
@@ -75,5 +75,7 @@
 	TECHWEB_POINT_TYPE_DISCOVERY = "Discovery Research",\
 	TECHWEB_POINT_TYPE_NANITES = "Nanite Research"\
 	)
-
+//MonkeStation Edit: Passive Discovery Research
+GLOBAL_VAR_INIT(passive_discovery_research, 16)
```

Comment:
Can go in its own modular folder

Reply:
In this case, I think it may be best to keep it within the main defines.
The techweb and research system code is a complete nightmare, so this is one place where having all of the defines in one spot will be better than modularizing it.

Reply:
Aye, fair enough